README.md

```diff
@@ -1,22 +1,52 @@
 # VirtoCommerce.Cart
-VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager. 
 
-# Documentation
-User guide:
+## Overview
 
-Developer guide:
+VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager.
 
-# Installation
+Shopping cart module manages customers accumulated list of items, calculates a total for the order, including shipping and handling (i.e., postage and packing) charges and the associated taxes.
+
+![Cart Module](docs/media/screen-cart-module.png)
+
+![Cart Module Info](docs/media/screen-cart-module-info.png)
+
+## Features
+
+1. Supports multiple carts
+1. Wishlist
+1. Grouping multiple carts to one order
+1. Anonymous carts
+1. Stock reservation
+1. Multiple payments methods
+1. Create new cart from orders history
+
+The main purpose of the cart module is to save and store the shopping cart data and changes on VC Manager side.
```

Comment:
The main purpose of the Cart module is to implement customer shopping cart management in VC eCommerce solution. It encapsulates data persistence, management services and exposes REST API endpoints.